README.md

```diff
@@ -40,6 +40,10 @@ Check out the [docs](https://docs.luthersystems.com).
 
 *IMPORTANT:* Place your license key in `.luther-license.yaml` in the project root directory. (This is a change from previous versions, in order to support parallel Docker containers.)
 
+For devs using hardware that utilizes the Apple M1 chip, you may encounter an issue getting the sandbox environment up and running.
+To resolve this, ensure you have [Rosetta 2](https://support.apple.com/en-us/HT211861) installed on your computer. This can be done by installing
+any software that won't naturally run on the M1 chip (e.g. Zoom) and should appear as an additional prompt during the installation process.
```

Comment:
As mentioned in
https://derflounder.wordpress.com/2020/11/17/installing-rosetta-2-on-apple-silicon-macs/
it should be possible to do this directly, with
```
/usr/sbin/softwareupdate --install-rosetta
```
This seems like a better suggestion.